### PR TITLE
Optimize setup and future queries by creating core table

### LINKED ISSUE #15: Create "Core" Table Once and Then Never Again
We only need to create the `SPLIT_TEXT_TABLE` once and then never again. This should be a "Core" table that is created on initialize. This will greatly improve performance because then the WordCount and EmojiCount (and I imagine a word by day chart) tables can just read from this core table, which is created only once.

The core table could also already exclude stopWords, unicode characters, etc, so we don't have to run those WHERE clauses multiple times.

### PR DESCRIPTION
**Pull request by alexdanilowicz**
## Changes

Another "refactor": instead of creating tables for each chart, we should just query. There is no reason to create new tables unless the query is intense. If it's simple SELECTs/JOINs then we should just do what SQL is meant for and query directly.

Resolves #15

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
-    ".eslintrc": "jsonc",
-    ".prettierrc": "jsonc",
-    ".eslintignore": "ignore"
-  },
+    "eslint.workingDirectories": [
+        {
+            "directory": "electron",
+            "changeProcessCWD": true
+        }
+    ],
```
Comment (alexdanilowicz) [inline .vscode/settings.json:7]: This is only real non-whitespace change.

File: electron/src/chatBro/queries/TopFriends/TopFriends.ts [hunk 2]
```diff
@@ -1,15 +1,34 @@
...
+const FROM_QUERY = `
+  SELECT
+    COUNT(*) as ${Columns.COUNT},
+    h.id as ${Columns.PHONE_NUMBER},
+    COALESCE(h.contact_name, h.id) as ${Columns.FRIEND},
+    is_from_me as ${Columns.IS_FROM_ME}
+  FROM message
+    JOIN handle h
+    ON h.ROWID = handle_id
+    -- NOTE: Excludes group chats
```
Comment (alexdanilowicz) [inline electron/src/chatBro/queries/TopFriends/TopFriends.ts:21]: We would change this to be filters like we have the WordOrEmoji query set up

File: electron/src/chatBro/queries/WordCount/WordCount.ts [hunk 3]
```diff
@@ -19,28 +20,40 @@ function wordFilter(opts: WordCountTypes.Options): string | undefined {
...
+}
+
+function isEmojiFilter(opts: WordCountTypes.Options): string | undefined {
+  if (opts.isEmoji === true) {
+    return `TRIM(${Columns.WORD}) IN (${emojis})`;
+  }
+  if (opts.isEmoji === false) {
+    return `TRIM(${Columns.WORD}) NOT IN (${emojis})`;
+  }
+  return undefined;
```
Comment (alexdanilowicz) [inline electron/src/chatBro/queries/WordCount/WordCount.ts:33]: I prefer readability above all else, but if anyone has any recs on how to make this cleaner, lmk

File: electron/src/components/charts/WordCount.tsx [hunk 4]
```diff
@@ -1,66 +0,0 @@
-import React, { useState, useEffect } from 'react';
-import { Bar } from 'react-chartjs-2';
-import * as sqlite3 from 'sqlite3';
-import log from 'electron-log';
-import { ChatTableNames } from '../../tables';
-import * as chatBro from '../../chatBro';
-import interpolateColors from '../../utils/colors';
-import ChartLoader from '../loading/ChartLoader';
-
```
Comment (alexdanilowicz) [inline electron/src/components/charts/WordCount.tsx:9]: This file should have been deleted a long time ago. Drive-by change.

File: electron/src/chatBro/queries/WordCount/WordCount.ts (hunk 3 above)
Comment (ayc19) [reply electron/src/chatBro/queries/WordCount/WordCount.ts:33]: will this return undefined ever get hit? Looks like if first if statement is false, doesn't that guarantee the second one is true

Comment (alexdanilowicz) [reply electron/src/chatBro/queries/WordCount/WordCount.ts:33]: You know, I had a shower thought about that ... I was mimicking this off of the other filters we have. 

I will change this.
